tests/unit_tests/test_workflow.py

```diff
@@ -218,6 +218,7 @@ def test_calibration_manager_save_load():
         "fwhm_rt": 5,
         "fwhm_mobility": 0.01,
         "score_cutoff": 50,
+        "gradient_length": 1000,
```

Comment:
I think we can get away without modifying the config during processing to avoid chaos :) so this shouldn't be necessary.